src/mame/drivers/midqslvr.cpp

```diff
@@ -2,25 +2,254 @@
+|                 U20                                                                        |
+|JP11   U21                                                                                  |
+|                                                                                            |
+|                                                          U6          S2        S1          |
+|                JP20                                                                        |
+|                                                                                            |
+|JP16      U12                      U16        U15                                           |
+|                                                                                            |
+|                                                                                      U11   |
+|      JP17         JP19           JP18              JP13             JP14           JP15    |
+|--------------------------------------------------------------------------------------------|
+
+Notes:
+    J1: Video connector from/to video card
+    J2: USB port (not used)
+    JP1: 3 pin jumper: Blue video level, *Open: high, 1-2: low, 2-3: high
+    JP2: 3 pin jumper: Green video level, *Open: high, 1-2: low, 2-3: high
+    JP3: 3 pin jumper: Red video level, *Open: high, 1-2: low, 2-3: high
+    JP4: 3 pin jumper: Video sync polarity, Open: pos, *1-2: neg, 2-3: pos
+    JP5: Alternate video connector from/to video card
+    JP6: Video output signal to monitor, EGA resolution
+    JP9: Power connector
+    JP10: 20 pin ribbon cable connector to wheel driver board
+    JP11: 20 pin ribbon cable connector, not used
+    JP12-14: connectors, not used
+    JP15: 14 pin connector for analog controls/potentiometers
+    JP16: 15 pin connector for gameplay inputs
+    JP17: connector, not used
+    JP18: 14 pin connector for coin, service, volume and test inputs
+    JP19: connector, not used
+    JP20: connector for development use, not used
+    JP21: 2 pin connector for watchdog reset
+    JP22: 9 pin connector for coin meter
+    JP23: Alternate RS232 port
+    JP24: connector, not used
+    P1: DB9 RS-232 port to computer
+	Q4: ULN2064B Darlington Transistor
+	S1: Dip Switches (8)
+         S1-7: *Off: UART, On: USB
+         S1-8: Off: Watchdog Disabled, *On: Watchdog Enabled
+	S2: Dip Switches (8), all set to "off"
+	U1: LS85A Logic Gate
+    U2-3: EL244CS Amplifier
+    U5: MAX707CSA Supervisory Circuit
+    U6: Motorola MC74HC273A Octal D Flip-Flop (LS273 based)
+    U11: ADC0834 Serial I/O Converter
+    U12-15: HC541 Octal Buffer
+    U17: Atmel 24C01A Serial EEPROM
+    U18: CY7C63513-PVC 8-bit RISC Microcontroller
+    U19: DS14185WM RS-232 Interface IC
+    U20: PC16550DV UART Interface IC
+    U21: Oscilator 3.6864 MHz
+    U22: HC04 Hex Inverter
+    Y1: Crystal/XTAL 6.000 MHz
+
+YOU CAN'T HAVE IT
```

Comment:
There should be a note here indicating that these are markings on the PCB, as otherwise they look strangely out of place.

Reply:
Removed.